src/main/java/com/ecore/roles/web/RolesApi.java

```diff
@@ -0,0 +1,19 @@
+package com.ecore.roles.web;
+
+import com.ecore.roles.web.dto.RoleDto;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface RolesApi {
+
+    ResponseEntity<RoleDto> createRole(
+            RoleDto role);
+
+    ResponseEntity<List<RoleDto>> getRoles();
+
+    ResponseEntity<RoleDto> getRole(
```

Comment:
It could be just called "get" or "getById", because it's inside of the role api.